### PR TITLE
minor : LineWrapping for few lines in releasenotes.xml

### PR DESCRIPTION
**Pull request by AmitKumarDeoghoria**
LineWrapping for few lines in releasenotes.xml

### CODE REVIEW

**Comment by romani** [comment]
Please drop first commit from this PR, just keep minor commit

File: src/xdocs/releasenotes.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
       <p>New:</p>
         <ul>
           <li>
-            HideUtilityClassConstructor - Add option to skip validation based on list of annotations.
+            HideUtilityClassConstructor - Option to skip validation for selected annotations.
```
Comment (romani) [inline src/xdocs/releasenotes.xml:26]: Can we just go line wrapping, no changes in wording

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/releasenotes.xml (hunk 1 above)
Comment (AmitKumarDeoghoria) [reply src/xdocs/releasenotes.xml]: Done.

**Review by romani — APPROVED** [review]
Thanks a lot

**Pull request merged**